terraform/labs/lambda_grant_user_access/terragrunt.hcl

```diff
@@ -12,7 +12,9 @@ include {
 }
 
 inputs = {
-  lambda_function_name = local.product
-  timeout_in_seconds   = 900
-  tags                 = { Git_Project = "https://github.com/hmrc/aws-${local.product}" }
+  lambda_function_name  = local.product
+  timeout_in_seconds    = 900
+  sns_topic_arn         = "arn:aws:sns:eu-west-2:304923144821:psec-1933-grant-user-access"
```

Comment:
Did you intend to have the jira number in here or is this temporary?

Reply:
I need it for testing purposes only for now. This SNS topic is in Sandbox Account, which gives me a place to test. It also replicates the prod setup. i.e. the grant-user-user-access lambda and the SNS topic are in different accounts